### ISSUE TITLE
Datenexport für GML-erzeugung

### ISSUE THREAD

**Issue by petershr01** [bstroebl/xplanPostGIS#15]
Nächste Fragestellung:
Die derzeit einzige freie Anwendung um XPlan-Dateien zu erstellt, GML-Toolbox, benötigt Shape-Dateien. Bei der Verwendung der PostgreSQL/PostGIS Fachschale sind viele Sachdaten in verschiedenen DB-Tabellen abgelegt.
Frage wäre jetzt, gibt es wenigstens eine DB-Tabelle die zumindest alle Pflichtfelder enthält um diese dann nach Shape zu exportieren?
Oder müsste man einen VIEW erstellen, in dem dann alle wichtigen Datenfelder enthalten sind.

Nur als Bemerkung:
Generell ist die Erzeugung von XPlan-Konformen GML-Dateien derzeit das größte Problem.

**Comment by bstroebl** [comment]
Ja, ist mir bewusst, dass niemand wirklich sinnvoll XPlan-GML schreiben kann; leider kann man n-zu-m-Beziehungen nicht in eine "flache" Shapedatei packen (und also auch nicht in einen View). Hierfür ist ein deutlich komplexerer Ansatz nötig.

**Comment by petershr01** [comment]
Ok, danke für die Antwort. Hatte gehofft, dass ich was übersehen habe. Vielleicht tut sich zu diesem Thema in den nächsten Monaten noch was.
